Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN docker-php-ext-install mcrypt \
       intl \
       pgsql
 
+RUN pecl install -f memcache
```

Comment:
okay it's not supported apparently :)

then, could we do:
`RUN cd /tmp && git clone https://github.com/php-memcached-dev/php-memcached.git && cd php-memcached && php-ize && make && make install` ?

pecl is not a nice dependency to have afaik, and there's no guarantee that it's maintained properly. Maybe @silvanu also has some experience with this, from the Unitt cookbooks.